Source/Core/Core/Movie.h

```diff
@@ -36,10 +36,11 @@ struct ControllerState
   bool Start : 1, A : 1, B : 1, X : 1, Y : 1, Z : 1;  // Binary buttons, 6 bits
   bool DPadUp : 1, DPadDown : 1,                      // Binary D-Pad buttons, 4 bits
       DPadLeft : 1, DPadRight : 1;
-  bool L : 1, R : 1;  // Binary triggers, 2 bits
-  bool disc : 1;      // Checks for disc being changed
-  bool reset : 1;     // Console reset button
-  bool reserved : 2;  // Reserved bits used for padding, 2 bits
+  bool L : 1, R : 1;     // Binary triggers, 2 bits
+  bool disc : 1;         // Checks for disc being changed
+  bool reset : 1;        // Console reset button
+  bool IsConnected : 1;  // Should controller be treated as connected
```

Comment:
Then you would be inconsistent with `disc` and `reset`. Since the naming isn't consistent across the struct, it's best to go with what our style guide says.